src/components/file-browser/FileBrowser.ts

```diff
@@ -19,6 +19,11 @@ const FileBrowser = ({ provider, title, workspace }: FileBrowserProps) => {
 
   const [focusedFile, setFocusedFile] = useState<FileBrowserFile | null>(null)
 
+  const [selectedFiles, setSelectedFiles] = useState<{ [path: string]: FileBrowserFile }>({})
```

Comment:
Could you explain the syntax for `{ [path: string]: FileBrowserFile }`?

Reply:
It's an "index signature": https://www.typescriptlang.org/docs/handbook/2/objects.html#index-signatures

It says that this state variable contains an object with string keys and FileBrowserFile values.

Reply:
I see. Thanks!